dissect/target/plugins/browsers/browser.py

```diff
@@ -95,6 +104,62 @@ def _func(self, func_name: str):
             except Exception:
                 self.target.log.exception("Failed to execute browser plugin: %s.%s", plugin_name._name, func_name)
 
+    def check_compatible(self) -> bool:
+        """Perform a compatibility check with the target.
+        This function checks if any of the supported browser plugins
+        can be used. Otherwise it should raise an ``UnsupportedPluginError``.
+        Raises:
+            UnsupportedPluginError: If the plugin could not be loaded.
+        """
+        if not len(self._plugins):
+            raise UnsupportedPluginError("No compatible browser plugins found")
+
+    @export(record=BrowserDownloadRecord)
+    def downloads(self):
+        """Return browser download records from all browsers installed and supported.
+
+        Yields:
+            BrowserDownloadRecord with the following fieds:
```

Comment:
```suggestion
        Yields BrowserDownloadRecord with the following fieds:
```
This is more in line with the other docstrings.